NineChronicles.Standalone/RPCContext.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Immutable;
+using Libplanet;
+
+namespace NineChronicles.Standalone
+{
+    public class RpcContext
+    {
+        public ImmutableList<Address> AddressesToSubscribe = ImmutableList<Address>.Empty;
```

Comment:
I guess it would be better if we use `ImmutableHashSet<>` instead of `ImmutableList<>`.